gigantum/gigantum-client
# Adds error handling and ui blocking to repository creation.

## Pull Request for Issue #1674

**Describe changes introduced by this PR (Bugs fixed, features added, docs updated, etc.)**:

**_Frontend_**

Adds error handling when repository name cannot be verified. Block repository creation when backup mode has been 
detected.  

**_Backend_**

**Please check if the PR fulfills these requirements**

  - PR is set to merge to integration
  - PR title is descriptive, as this becomes the commit message on squash and merge.
  - Any related issues are added to the extended commit message during squash and merge.
  - Tests have been added (for bug fixes/features)
  - Docs have been added/updated (for bug fixes/features)

**QA Test Steps and Exit Criteria**:
1. Load ui
2. switch to back up mode
3. test project creation, should fail with backup in progress message.
4. reload and verify that project creation is blocked

repeat 1-4 for datasets
